front/src/applications/operationalStudies/helpers/TrainSimulationLazyLoader.ts

```diff
@@ -138,7 +138,7 @@ export default class TrainSimulationLazyLoader {
     }
     for (const [rawId, rawSummary] of Object.entries(rawPacedTrainSummaries)) {
       const id = formatEditoastIdToPacedTrainId(Number(rawId));
-      rawSummaries.set(id, rawSummary);
+      rawSummaries.set(id, rawSummary.paced_train);
```

Comment:
If you could add that also thank you
```suggestion
      // TODO exceptions : adapt this to handle exceptions summaries
      rawSummaries.set(id, rawSummary.paced_train);
```